kieker-monitoring/gradle.properties

```diff
@@ -1,8 +1,8 @@
 checkstyleErrorThreshold = 0
-checkstyleWarningThreshold = 5
+checkstyleWarningThreshold = 8
```

Comment:
I think the pull request description should indicate if and why thresholds are increased.